elixir/apps/web/lib/web/live/sites/new_token.ex

```diff
@@ -236,20 +257,29 @@ defmodule Web.Sites.NewToken do
             echo "Unsupported architecture"; \\
             exit 1 ;; \\
         esac; \\
-        wget -O /usr/local/bin/firezone-gateway $bin_url; \\
-        chmod +x /usr/local/bin/firezone-gateway; \\
-        mkdir -p /etc/firezone; \\
-        chmod 0755 /etc/firezone; \\
-        iptables-nft -A FORWARD -i tun-firezone -j ACCEPT; \\
-        iptables-nft -A FORWARD -o tun-firezone -j ACCEPT; \\
-        iptables-nft -t nat -A POSTROUTING -o e+ -j MASQUERADE; \\
-        ip6tables-nft -A FORWARD -i tun-firezone -j ACCEPT; \\
-        ip6tables-nft -A FORWARD -o tun-firezone -j ACCEPT; \\
-        ip6tables-nft -t nat -A POSTROUTING -o e+ -j MASQUERADE; \\
+        curl -Ls $bin_url -o /usr/local/bin/firezone-gateway; \\
+        chgrp firezone /usr/local/bin/firezone-gateway; \\
+        chmod 0750 /usr/local/bin/firezone-gateway; \\
```

Comment:
BTW, on some security-hardened systems there is no `/usr/local/bin` and you can't write to it either:
```
andrew@web-zqg3 ~ $ sudo mkdir -p /usr/local/bin
mkdir: cannot create directory ‘/usr/local/bin’: Read-only file system
```
but we can ignore it for some time.

Reply:
Eesh, you're right. We'll tackle that probably in https://github.com/firezone/firezone/issues/2944